plotly/plotly/plotly.py

```diff
@@ -24,11 +24,9 @@
 import six.moves
 from requests.compat import json as _json
 
-from plotly import exceptions, tools, utils, files
+from plotly import exceptions, files, session, tools, utils
```

Comment:
I'm just curious, why did you switch up the order like this? Is it because `tools` is moving into a different location for `2.0.0.` and therefore its priority is being hierarchically diminished? (I don't know the proper jargon for what I'm describing...)